### PR TITLE
Moving gemspec and files to root directory

### PR DESCRIPTION
**Pull request by veronicagg**
Moving gemspec and files to root directory, updating readme and travis file content.

### CODE REVIEW

File: service_management/azure/test/fixtures/management_certificate.pem [hunk 1]
```diff
@@ -1,55 +0,0 @@
-Bag Attributes
```
Comment (vishrutshah) [inline service_management/azure/test/fixtures/management_certificate.pem]: Don't we need this file anymore or git doesn't show as moved?

File: service_management/azure/test/fixtures/management_certificate.publishsettings [hunk 2]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (vishrutshah) [inline service_management/azure/test/fixtures/management_certificate.publishsettings]: Don't we need this file anymore or git doesn't show as moved?

File: service_management/azure/test/fixtures/management_certificate.pem (hunk 1 above)
Comment (veronicagg) [inline service_management/azure/test/fixtures/management_certificate.pem]: Thanks @vishrutshah. Looks like travis failed due to that, adding these files back.

**Comment by vishrutshah** [comment]
@veronicagg  Changes looks good to go! Thanks.

**Pull request merged**